Mathlib/FieldTheory/Galois/Basic.lean

```diff
@@ -205,6 +209,17 @@ lemma fixingSubgroup_anti : Antitone (IntermediateField.fixingSubgroup (F := F)
   rw [← le_iff_le]
   exact le_trans h ((le_iff_le _ _).mpr (le_refl K'.fixingSubgroup))
 
+@[simp] lemma mem_fixingSubgroup_iff (σ) : σ ∈ fixingSubgroup K ↔ ∀ x ∈ K, σ x = x :=
+  _root_.mem_fixingSubgroup_iff _
```

Comment:
I wonder whether the `_root_` here is an indication that the namespaced `mem_fixingSubgroup_iff` should be `protected` instead.